openedx/features/philu_courseware/helpers.py

```diff
@@ -54,11 +54,20 @@ def get_competency_assessments_score(user, chapter_id):
     pre_assessment_score = post_assessment_score = 0
     attempted_pre_assessments = attempted_post_assessments = 0
 
-    COMPETENCY_ASSESSMENT_RECORD_QUERY_FORMAT = 'SELECT MAX(id) AS id, COUNT(assessment_type) AS assessments_count, ' \
-                                      'assessment_type, correctness FROM philu_courseware_competencyassessmentrecord' \
-                                      ' WHERE id IN (SELECT MAX(id) FROM philu_courseware_competencyassessmentrecord' \
-                                      ' WHERE chapter_id = "{chapter_id}" and user_id = {user_id} ' \
-                                      'GROUP BY problem_id) GROUP BY correctness, assessment_type'
+    COMPETENCY_ASSESSMENT_RECORD_QUERY_FORMAT = """
```

Comment:
Can we please add unit tests for this method as well? I know there is no existing unit test for this method but as we have some buffer, we can write.

Reply:
this method is called in three other tests, so I think it's unnecessary.

Reply:
Because this is a separate unit, that's why I think we should have its separate unit test. This separate helper can be used anywhere in the system and the one who is using it won't write it's unit test. They will just mock it and use it.

Reply:
@ahmed-faraz46 in case of any break in this function will cause exception for other unit tests, that is why I think that is totally unnecessary.